forest/benchmarking/tests/test_distance_measures.py

```diff
@@ -231,6 +231,76 @@ def _gate_to_choi(gate):
     dnorm = dm.diamond_norm(choi0, choi1)
     assert np.isclose(dnorm, np.sqrt(2), rtol=0.01)
 
+def test_watrous_bounds():
+    # Test cases borrowed from qutip,
+    # https://github.com/qutip/qutip/blob/master/qutip/tests/test_metrics.py
+    # which were in turn generated using QuantumUtils for MATLAB
+    # (https://goo.gl/oWXhO9) by Christopher Granade
+
+    _I = np.asarray([[1, 0], [0, 1]])
+    _X = np.asarray([[0, 1], [1, 0]])
+    _Y = np.asarray([[0, -1.0j], [1.0j, 0]])
+    _H = np.asarray([[1, 1], [1, -1]]) / np.sqrt(2)
+
+    def _gate_to_superop(gate):
```

Comment:
It looks like there is a lot of shared boilerplate between this and the `test_diamond_norm` UT, and additionally some of the functions seem very similar to some of the utilities in `superop_conversion.py`. Would it be worth consolidating some of the boilerplate and/or using some of our own utilities?

Reply:
I was going to make an issue for that. Gavin wrote `diamond_norm` before we wrote the `superop_conversion` module.

Reply:
Yeah, I noticed that as well. TBH, we also need to revisit how to implemented the diamond norm -- there are better SDPs for it.

Reply:
Let's fix this in a separate PR

Reply:
@marcusps or @joshcombes could we open an issue?

Reply:
illl open an issue now